hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/KeyDeletingService.java

```diff
@@ -323,8 +323,8 @@ public BackgroundTaskResult call() {
               SnapshotUtils.getSnapshotInfo(getOzoneManager(), snapshotChainManager, snapshotId);
           if (snapInfo != null) {
             if (snapInfo.isDeepCleaned()) {
-              LOG.info("Snapshot {} has already been deep cleaned. Skipping the snapshot in this iteration.",
-                  snapInfo.getSnapshotId());
+              LOG.info("Snapshot '{}' ({}) has already been deep cleaned. Skipping the snapshot in this iteration. ",
```

Comment:
nit. oops sorry i should have caught this extra space earlier

```suggestion
              LOG.info("Snapshot '{}' ({}) has already been deep cleaned. Skipping the snapshot in this iteration.",
```

Reply:
done